test/retry.test.ts

```diff
@@ -228,7 +228,7 @@ describe("Automatic Retries", function () {
 
     expect(caught).toEqual(testStatuses.length);
     expect(octokit.__requestLog).toStrictEqual(
-      testStatuses.map((x) => "START GET /route")
+      testStatuses.map((_) => "START GET /route")
```

Comment:
I would instead remove the argument all together

Reply:
Done!